Prep/Source/Alignment/ICPRigid3DImageRegistration/source/CMakeLists.txt

```diff
@@ -4,3 +4,7 @@ cmake_minimum_required(VERSION 2.8)
 ADD_EXECUTABLE(ICPRigid3DImageRegistration ICPRigid3DImageRegistration.cxx)
 TARGET_LINK_LIBRARIES(ICPRigid3DImageRegistration ${VTK_LIBRARIES} ${ITK_LIBRARIES} OptionParser)
 INSTALL(TARGETS ICPRigid3DImageRegistration  RUNTIME DESTINATION bin)
+
+ADD_EXECUTABLE(MatrixRigid3DImageRegistration MatrixRigid3DImageRegistration.cxx)
```

Comment:
- This tool should be renamed to be descriptive of its function. If it is only applying a given transformation matrix to an image, you could use ApplyRigid3DTransformationToImage
- We could also add a flag to allow the user to apply the transformation on segmentations. Please submit an issue for this and we can work it out later.

Reply:
I have renamed the cxx file as well as CMAkeLists.txt